CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 otj-metrics
 ===========
 
+5.2.4
+-----
+* "otj-metrics-actuator" now has ability to breakdown httpServerRequests by referring service. You need to opt-in for this feature by specifying list of services to track. Please see `otj-mvc-demo` project for example.  
```

Comment:
can you elaborate in the README what needs to be toggled on and the bean injection. That shouldn't just be a demo